docs/book/src/topics/workload-identity.md

```diff
@@ -119,26 +120,6 @@ export AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE="default"
 clusterctl generate cluster azwi-quickstart --kubernetes-version v1.27.3  --worker-machine-count=3 > azwi-quickstart.yaml
 ```
 
-- Edit the generated `azwi-quickstart.yaml` to make the following changes for
-  workload identity to the `AzureClusterIdentity` object.
-  - Change the type to `WorkloadIdentity`.
-  - Remove the `clientSecret` spec.
-
-The AzureClusterIdentity specification should look like the following.
-```yaml
-apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-kind: AzureClusterIdentity
-metadata:
-  name: cluster-identity
-spec:
-  type: WorkloadIdentity
-  allowedNamespaces:
-    list:
-    - <cluster-namespace>
-  tenantID: <your-tenant-id>
-  clientID: <your-client-id>
-```
```

Comment:
Just wanted to make sure: does changing the flavor templates also change the output of `clusterctl generate`, which is why we no longer need this part of the doc?